apps/web/src/utils/proofs/sybil_resistance.ts

```diff
@@ -140,7 +144,7 @@ export async function sybilResistantUsernameSigning(
     const hasPreviouslyRegistered = await hasRegisteredWithDiscount(linkedAddresses, chainId);
     // if any linked address registered previously return an error
     if (hasPreviouslyRegistered) {
-      throw new Error('You have already claimed a username with a different address (onchain).');
+      throw new Error('You have already claimed a discounted basename (onchain).');
```

Comment:
the connected address is included in `getLinkedAddresses` so its possible that the discounted name was already claimed with the connected address, not a different one